Apps/CocoaPods-FCM/src/Extensions/BindingExtensions.swift

```diff
@@ -16,7 +16,7 @@ extension Binding where Value == Double {
 
 extension Binding where Value == Int {
     func toStringBinding() -> Binding<String> {
-        cast(to: { String($0) }, from: { Int($0)! })
+        cast(to: { String($0) }, from: { $0 == "" ? 0 : Int($0)! })
```

Comment:
similarly, there are a lot of forced unwrapping in other files, i didn't comment on all of them but maybe something we can avoid to avoid crash on edge cases?

Reply:
For sample apps, I am OK with using forced unwrapping where the non-forced unwrapping solution adds complexity that is not valuable to the app. I say this because sometimes a non-forced unwrap solution feels like overkill for an internal project like a sample app.